R/espn_starters.R

```diff
@@ -67,14 +86,7 @@ ff_starters.espn_conn <- function(conn, weeks = 1:17, ...) {
   return(starters)
 }
 
-.espn_week_checkmax <- function(conn) {
-  url_query <- glue::glue(
-    "https://fantasy.espn.com/apis/v3/games/ffl/seasons/",
-    "{conn$season}/segments/0/leagues/{conn$league_id}",
-    "?scoringPeriodId=0&view=mSettings"
-  )
-
-  settings <- espn_getendpoint_raw(conn, url_query)
+.espn_week_checkmax <- function(settings) {
```

Comment:
took the leisure of editing this function to take in the `settings` list instead of the raw connection. the proposed `.espn_week_matchup_periods_mapping()` also needs elements in the settings list. This refactor avoids hitting the settings endpoint more than once.